packages/trimerge-sync/src/TrimergeClient.ts

```diff
@@ -273,14 +273,11 @@ export class TrimergeClient<
         this.updateSyncState({ localRead: 'ready' });
         break;
       case 'error':
-        if (event.code === 'internal' || event.code === 'unauthorized') {
-          if (remoteOrigin) {
-            this.emitError('remote', new ErrorEventError(event));
-            this.updateSyncState({ remoteRead: 'error' });
-          } else {
-            this.emitError('local-store', new ErrorEventError(event));
-            this.updateSyncState({ localRead: 'error' });
-          }
+        if (remoteOrigin) {
+          this.emitError('remote', new ErrorEventError(event));
```

Comment:
There is an explicit `remote-state` event so we remove this since the error may not be related to reading.